src/adr/0002-make-taskmanager-distributed.md

```diff
@@ -0,0 +1,25 @@
+# 2. Make TaskManager Distributed
+
+Date: 2019-10-02
+
+## Status
+
+Accepted (lazy consensus)
```

Comment:
Actually we can not consider lazy consensus applies for a pr on linagora repository, not advertised on the msiling list.

Reply:
Agree, but we just need these ADR to be posted on the mailing list and/or a JIRA ticket (after approval of this PR, no?). But if you prefer, I can change all status to "Proposed"... and wait for the post to merge...

Reply:
You mean, this is a pre-proposal step?

Reply:
given the definition of status (see apache#169) we decided during this PR, yes.